src/to-have-focus.js

```diff
@@ -13,9 +13,9 @@ export function toHaveFocus(element) {
           '',
         ),
         '',
-        'Expected',
+        'Expected element with focus:',
         `  ${this.utils.printExpected(element)}`,
-        'Received:',
+        'Received element without focus:',
```

Comment:
The element printed on this part of the error message is not an element without focus. It is the element that currently has the focus. So this proposed new heading is incorrect.

Reply:
Ahh gotcha.  Updated the error message to better reflect